jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyHomeForker.java

```diff
@@ -74,6 +75,11 @@
      */
     protected String[] modules;
 
+    /*
+     * Optional jetty commands
+     */
+    protected String jettyOptions;
```

Comment:
Make this `protected String[] jettyOptions` and don't split on space.

Reply:
but jvmArgs will be different? 
plugin such surefire (`argLine`) use String to avoid too big xml :) 
I vote for `String`  and using command line to pass parameters will be more complicated.

Reply:
Then the split needs to be more advanced.

* not just on space.
* It should take into account escaped spaces.
* It should also honor quoted sequences.
* It should eliminate parts that are functionally blank (all whitespace)

I'm not convinced that supporting `argLine` is an sufficient argument for this.

Reply:
The `jvmArgs` has been a single single string argument for years so far with no reported problems. The surefire plugin also uses just a single string for its jvm arg line. I don't see what's different about the jetty command args that would warrant making it so much more verbose to configure in the poms. Plus I think the `jvmArgs` and the new `jettyOptions` should both be the same: changing to `String[]` would be a breaking change for the `jvmArgs`.